meta-mender-core/recipes-mender/mender-shell/mender-shell.bb

```diff
@@ -14,16 +14,22 @@ PV = "0.1+git${SRCPV}"
 LICENSE = "Apache-2.0 & BSD-2-Clause & BSD-3-Clause & ISC & MIT"
 LIC_FILES_CHKSUM = "file://src/${GO_IMPORT}/LICENSE;md5=7fd64609fe1bce47db0e8f6e3cc6a11d"
 
-inherit go-mod
-inherit go-ptest
-
-DEPENDS_append = " pkgconfig-native glib-2.0"
+DEPENDS_append = " glib-2.0"
 RDEPENDS_${PN} = "glib-2.0"
 
+SYSTEMD_AUTO_ENABLE ?= "enable"
+
+inherit go
+inherit go-ptest
+inherit pkgconfig
+inherit systemd
+
 GO_IMPORT = "github.com/mendersoftware/mender-shell"
 
 do_compile() {
-    oe_runmake V=1
+    oe_runmake \
+        -C ${B}/src/${GO_IMPORT} \
```

Comment:
Just curious:
Why is this needed, like it was working for me previously?

Reply:
I don't know exactly why, I haven't inspected the bitbake code this time. But with `inherint go-mod` this was not necessary, while with `inherit go` turned out necessary. Also note that it was already used for `do_install` below.